test/test_deploy_templates.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import pytest
+from pathlib import Path
+
+from container_ci_suite.openshift import OpenShiftAPI
+from container_ci_suite.utils import get_service_image, check_variables
+test_dir = Path(os.path.abspath(os.path.dirname(__file__)))
```

Comment:
You use all upper case names for constant and this is also constant.

Reply:
In case it is used, then UPPER_CASED